portfolio/src/main/webapp/index.html

```diff
@@ -33,22 +34,22 @@ <h2>External profiles</h2>
     </ul>       
     <h2>Picture Gallery</h2>
     <div class="gallery" id="gallery">
-      <div>
+      <div class="mb-3 pics animation all 2">
```

Comment:
Do the `pics`, `animation`, etc classes refer to something in bootstrap? I found a similar example [here](https://www.edureka.co/blog/bootstrap-gallery/) where they use these classes, but they define them in their CSS file directly.

If you haven't defined them in your CSS and they aren't defined in the bootstrap CSS then I don't think these are doing anything.

Reply:
I thought they weren't doing anything either originally so I removed them but they actually create the border around the pictures. They are bootstrap classes.

Reply:
I think the `mb-3` class is probably adding the border ([its in the list of bootstrap classes here, and it adds a bottom margin](https://www.w3schools.com/bootstrap4/bootstrap_ref_all_classes.asp)). Could you try keeping that and removing the pics, animation, etc, and seeing if something changes?

Reply:
Ah yes, this worked. So now the class is just simplified to mb-3. Thanks!